src/core/data-cite.js

```diff
@@ -52,10 +54,16 @@ function requestLookup(conf) {
     }
     switch (elem.localName) {
       case "a": {
+        if (elem.text === "") {
+          elem.text = title;
+        }
         elem.href = href;
         break;
       }
       case "dfn": {
+        if (elem.text === "") {
+          wrapInner(elem, hyperHTML`<a href="${href}">${title}</a>`);
```

Comment:
Won’t this potentially wrap the element twice? See line 67

Reply:
Ahh, fixed that. 😄